Applications/VpView/vpViewCore.cxx

```diff
@@ -8526,56 +8467,56 @@ void vpViewCore::reactToExternalProcessFileChanged(QString changedFile)
 
   if (this->ExecutedExternalProcessMode == 1)  // Learning mode
     {
-    this->loadFilters(this->ExternalProcessOutputFile.toStdString().c_str());
+    this->loadFilters(qPrintable(this->ExternalProcessOutputFile));
     return;
     }
 
   // Copy the current project and update the scene element file
   static int memoryProjectCounter = 0;
   QScopedPointer<vpProject> project(new vpProject(this->CurrentProjectId++));
   project->CopyConfig(*this->Projects[this->SessionView->GetCurrentSession()]);
-  QString projectName(
-    this->Projects[this->SessionView->GetCurrentSession()]->Name.c_str());
+  QString projectName =
+    this->Projects[this->SessionView->GetCurrentSession()]->Name;
   projectName += QString(".%1").arg(memoryProjectCounter++);
-  project->Name = projectName.toStdString();
+  project->Name = projectName;
 
   // For now, this is specific to FSE
   if (this->ExecutedExternalProcessMode == 0)
     {
-    project->SceneElementsFile = this->ExternalProcessOutputFile.toStdString();
+    project->SceneElementsFile = this->ExternalProcessOutputFile;
     project->SetIsValid(project->SceneElementsFile, vpProject::FILE_EXIST);
     }
 
   // For now this is specific to normalcy anamoly
   if (this->ExecutedExternalProcessMode == 2)
     {
-    project->EventsFile = this->ExternalProcessOutputFile.toStdString();
+    project->EventsFile = this->ExternalProcessOutputFile;
     project->SetIsValid(project->EventsFile, vpProject::FILE_EXIST);
     }
 
   // For now this is specific to normalcy anamoly
   if (this->ExecutedExternalProcessMode == 3)
     {
-    project->ActivitiesFile = this->ExternalProcessOutputFile.toStdString();
+    project->ActivitiesFile = this->ExternalProcessOutputFile;
     project->SetIsValid(project->ActivitiesFile, vpProject::FILE_EXIST);
     }
 
   project->FrameNumberOffset = this->FrameNumberOffset;
 
   // If filters file is set, wipe out all existing filters before loading
-  if (!project->FiltersFile.empty())
+  if (!project->FiltersFile.isEmpty())
     {
     emit removeAllFilters();
     }
 
 #ifdef VISGUI_USE_VIDTK
```

Comment:
Seems VIDTK still required to open project files with this branch - what am I missing

Reply:
> Later, I'll have a very limited implementation of non-VidTK I/O.

...which would be less usable if we're still reliant on VidTK just to parse the project file. There's also no good excuse for that limitation; `QSettings` is perfectly capable of parsing project files, even if it is convenient to require a slightly altered syntax for some fields.